To: vim_dev@googlegroups.com
Subject: Patch 7.3.207
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.207
Problem:    Can't compile with MSVC with pentium4 and 64 bit.
Solution:   Only use SSE2 for 32 bit. (Mike Williams)
Files:      src/Make_mvc.mak


*** ../mercurial/vim73/src/Make_mvc.mak	2011-02-15 18:06:11.000000000 +0100
--- src/Make_mvc.mak	2011-06-04 11:59:38.000000000 +0200
***************
*** 409,416 ****
  CPUARG =
  !endif
  !else
! # VC8/9 only allows specifying SSE architecture
! !if "$(CPUNR)" == "pentium4"
  CPUARG = /arch:SSE2
  !endif
  !endif
--- 409,416 ----
  CPUARG =
  !endif
  !else
! # VC8/9/10 only allows specifying SSE architecture but only for 32bit
! !if "$(ASSEMBLY_ARCHITECTURE)" == "x86" && "$(CPUNR)" == "pentium4"
  CPUARG = /arch:SSE2
  !endif
  !endif
*** ../vim-7.3.206/src/version.c	2011-05-25 21:18:02.000000000 +0200
--- src/version.c	2011-06-12 20:30:30.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     207,
  /**/

-- 
"The question of whether computers can think is just like the question
of whether submarines can swim."      -- Edsger W. Dijkstra

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
